infra/gcp/lib.sh

```diff
@@ -287,6 +287,23 @@ function empower_group_for_gcb() {
         --role roles/serviceusage.serviceUsageConsumer
 }
 
+# Grant KMS admin privileges to a principal
+# $1: The GCP project
+# $2: The group email
+function empower_group_for_kms() {
+    if [ $# -lt 2 -o -z "$1" -o -z "$2" ]; then
```

Comment:
Should we make that stricter by doing something like the below?
```bash
if [ $# -ne 2 -o -z "$1" -o -z "$2" ]; then
```

I saw, that other funcs here allow additional arguments, but that might just be a c&p "issue" from cases where additional optional arguments are allowed (e.g. `empower_group_to_gcr` or `empower_gcr_admins`).

Reply:
this is probably a good idea, but I'm OK to let this PR go in and fix it up across the board.